assets/css/lib/vendor/bootstrap-ie8.css.map

```diff
@@ -1 +1 @@
5 {\n  margin-top: 48px !important;\n  margin-bottom: 48px !important;\n}\n.p-lg-1 {\n  padding: 4px !important;\n}\n.pt-lg-1 {\n  padding-top: 4px !important;\n}\n.pr-lg-1 {\n  padding-right: 4px !important;\n}\n.pb-lg-1 {\n  padding-bottom: 4px !important;\n}\n.pl-lg-1 {\n  padding-left: 4px !important;\n}\n.px-lg-1 {\n  padding-right: 4px !important;\n  padding-left: 4px !important;\n}\n.py-lg-1 {\n  padding-top: 4px !important;\n  padding-bottom: 4px !important;\n}\n.p-lg-2 {\n  padding: 8px !important;\n}\n.pt-lg-2 {\n  padding-top: 8px !important;\n}\n.pr-lg-2 {\n  padding-right: 8px !important;\n}\n.pb-lg-2 {\n  padding-bottom: 8px !important;\n}\n.pl-lg-2 {\n  padding-left: 8px !important;\n}\n.px-lg-2 {\n  padding-right: 8px !important;\n  padding-left: 8px !important;\n}\n.py-lg-2 {\n  padding-top: 8px !important;\n  padding-bottom: 8px !important;\n}\n.p-lg-3 {\n  padding: 16px !important;\n}\n.pt-lg-3 {\n  padding-top: 16px !important;\n}\n.pr-lg-3 {\n  padding-right: 16px !important;\n}\n.pb-lg-3 {\n  padding-bottom: 16px !important;\n}\n.pl-lg-3 {\n  padding-left: 16px !important;\n}\n.px-lg-3 {\n  padding-right: 16px !important;\n  padding-left: 16px !important;\n}\n.py-lg-3 {\n  padding-top: 16px !important;\n  padding-bottom: 16px !important;\n}\n.p-lg-4 {\n  padding: 24px !important;\n}\n.pt-lg-4 {\n  padding-top: 24px !important;\n}\n.pr-lg-4 {\n  padding-right: 24px !important;\n}\n.pb-lg-4 {\n  padding-bottom: 24px !important;\n}\n.pl-lg-4 {\n  padding-left: 24px !important;\n}\n.px-lg-4 {\n  padding-right: 24px !important;\n  padding-left: 24px !important;\n}\n.py-lg-4 {\n  padding-top: 24px !important;\n  padding-bottom: 24px !important;\n}\n.p-lg-5 {\n  padding: 48px !important;\n}\n.pt-lg-5 {\n  padding-top: 48px !important;\n}\n.pr-lg-5 {\n  padding-right: 48px !important;\n}\n.pb-lg-5 {\n  padding-bottom: 48px !important;\n}\n.pl-lg-5 {\n  padding-left: 48px !important;\n}\n.px-lg-5 {\n  padding-right: 48px !important;\n  padding-left: 48px !important;\n}\n.py-lg-5 {\n  padding-top: 48px !important;\n  padding-bottom: 48px !important;\n}\n.m-lg-auto {\n  margin: auto !important;\n}\n.mt-lg-auto {\n  margin-top: auto !important;\n}\n.mr-lg-auto {\n  margin-right: auto !important;\n}\n.mb-lg-auto {\n  margin-bottom: auto !important;\n}\n.ml-lg-auto {\n  margin-left: auto !important;\n}\n.mx-lg-auto {\n  margin-right: auto !important;\n  margin-left: auto !important;\n}\n.my-lg-auto {\n  margin-top: auto !important;\n  margin-bottom: auto !important;\n}\n.text-lg-left {\n  text-align: left !important;\n}\n.text-lg-right {\n  text-align: right !important;\n}\n.text-lg-center {\n  text-align: center !important;\n}\n\n.navbar-expand-sm .navbar-collapse,\n.navbar-expand-md .navbar-collapse,\n.navbar-expand-lg .navbar-collapse,\n.navbar-expand-xl .navbar-collapse,\n.navbar-expand .navbar-collapse {\n  display: block !important;\n}\n\n.navbar > .container,\n.navbar > .container-fluid,\n.nav,\n.navbar {\n  display: block;\n}\n"]}
```

Comment:
One of the things we tried to do with the localgov_base theme was remove frameworks, are you sure you want to add bootstrap back in here? There's nothing specifically wrong with it, just wanted to make sure you were using it for a specific reason.